src/redux/actions.js

```diff
@@ -19,8 +18,3 @@ export const listenToQueries = () => (
     });
   }
 );
-
-export const deleteVisible = (index) => ({
```

Comment:
Didn't read everything, but how do you delete the visible ones after x amount of time? do they stay in the DOM now? :(